resin/auth.py

```diff
@@ -119,6 +148,12 @@ def get_token(self):
         Raises:
             InvalidOption: if not logged in and there is no token in Settings.
 
+        Examples:
+            # If you are logged in.
```

Comment:
OK, I think I have confused you. What I see as "wrong" in the rendered example is not its indentation, but rather that it's rendered as 2 separate blocks. That's what I meant from the beginning, sorry for the confusion. It should be as simple a fix as adding a check in the docs generator for whether the line starts with ``#``. Try the change and see the generated output whether it worked.

Don't lose much more time on this if you can't get it to work -- the rest of the PR looks fine :)